stdlib/universe/integral.go

```diff
@@ -261,18 +261,25 @@ func (t *integralTransformation) Process(id execute.DatasetID, tbl flux.Table) e
 		}
 		switch in.points {
 		case 0:
+			// We had no data therefore the area under the curve is zero.
```

Comment:
A null value might make more sense here, applying the logic for a single point below. If there are no points, there's no basis for computing an area for a given range.

Not asking for this to be changed, but it doesn't seem quite right.

Reply:
Yeah a null value would make more sense, although it would be a breaking change. Maybe if this comes up again we can add a parameter to change the behavior.